packages/reports/app/styles/navi-reports/components/grouped-list.less

```diff
@@ -8,35 +8,24 @@
 
   &__group-header {
     cursor: pointer;
-    margin: 2px 0;
+    margin: 3px 0;
 
     &-content {
       font-family: @font-family-sans-serif; // override icon font
       text-transform: capitalize;
-    }
```

Comment:
we don't need `font-family: @font-family-sans-serif` anymore after removing the `fa` class ☝️